HW_6/Task2.py

```diff
@@ -0,0 +1,59 @@
+'''Добавьте в пакет, созданный на семинаре шахматный модуль. Внутри него напишите код, решающий 
+задачу о 8 ферзях. Известно, что на доске 8×8 можно расставить 8 ферзей так, чтобы они не били 
+друг друга. Вам дана расстановка 8 ферзей на доске, определите, есть ли среди них пара бьющих друг 
+друга. Программа получает на вход восемь пар чисел, каждое число от 1 до 8 - координаты 8 ферзей.
+Если ферзи не бьют друг друга верните истину, а если бьют - ложь.'''
+
+import random
+from copy import copy
+
+
+class FerziBoard:
+    def __init__(self, length: int):
+        self.length = length
+        self.board = []
+        self.all_variants = []
+        self._place_next_ferz(0)
+
+    def _can_we_place(self, row: int, num: int):
+        for i, value in enumerate(self.board):
+            if value == num or (i - row) == (value - num) or (i - row) == (num - value):
+                return False
+        return True
+
+    def _place_next_ferz(self, row: int):
+        if row == self.length:
+            self.all_variants.append(copy(self.board))
+        else:
+            for col in range(self.length):
+                if self._can_we_place(row, col):
+                    self.board.append(col)
```

Comment:
И тут 
```suggestion
                    self.board.append((row, col))
```
Теперь будет выводиться по  2 координаты.